test/java/seedu/addressbook/storage/StorageStub.java

```diff
@@ -0,0 +1,131 @@
+package seedu.addressbook.storage;
+
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.storage.jaxb.AdaptedAddressBook;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Marshaller;
+import javax.xml.bind.Unmarshaller;
+import java.io.*;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * Represents the file used to store address book data.
+ */
+public class StorageStub extends Storage{
```

Comment:
If `StorageStub` is used for `LogicTest` only, it can be quite minimal. None of the methods require a proper implementation. Stubs don't usually do any work.